packages/js/data/src/crud/actions.ts

```diff
@@ -109,7 +125,7 @@ export const createDispatchActions = ( {
 	const createItem = function* ( query: Partial< ItemQuery > ) {
 		try {
 			const item: Item = yield apiFetch( {
```

Comment:
I believe this will require the `idQuery: IdQuery,` as a first parameter as well right?
As I get the `Error: Not all URL parameters were replaced` when running this:
`wp.data.dispatch('wc/admin/products/attributes/terms').createProductAttributeTerm({parent_id: 2 }, { name: 'Small' });`